components/hab/install.ps1

```diff
@@ -203,13 +236,25 @@ Function Assert-Habitat($ident) {
     }
 }
 
+Function Test-UsePackagesChefio($version) {
+    # The $_patch may contain the /release string as well.
+    # This is fine because we only care about major/minor for this 
+    # comparison. 
+    $_major,$_minor,$_patch = $version -split ".",3,"SimpleMatch"
+    !$version -Or $version -ge 1 -Or $version -ge 89
```

Comment:
this seems wrong? I'm assuming you meant `!$version -Or $_major -ge 1 -Or $_minor -ge 89`. Personally I prefer `!$version -Or [Version]::new($_major, $_minor) -ge [Version]::new("0.89")`

Reply:
:man_facepalming: Yes, that is what it should be, will fix.  

Unfortunately, ` [Version]::new ` threw an error on 2008r2 so I had to revert to this approach

Reply:
```
PS C:\Users\Administrator\Desktop> .\install.ps1 -v 0.79.1
Installing Habitat 'hab' program
Method invocation failed because [System.Version] doesn't contain a method named 'new'.
At C:\Users\Administrator\Desktop\install.ps1:247 char:5
+     !$version -Or ([Version]::new($_major, $_minor) -ge [Version]::new("0.89"))
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MethodNotFound
```